openedx/edx-ora2
# revert: rollback changes from #2171 and #2181

## Description
We refactored the way we store content tag data while exporting the XBlock and some previous changes are not needed anymore. This reverts the following PRs:
- #2171
- #2181

## More information
- Related to https://github.com/openedx/modular-learning/issues/180
- Discussion here: https://github.com/openedx/edx-platform/pull/34270#discussion_r1508474276

Depends on:
- https://github.com/openedx/openedx-learning/pull/164
- https://github.com/openedx/edx-ora2/pull/2181

## Testing instructions
- Make sure the tests cover the various cases.

___
Private ref: [FAL-3621](https://tasks.opencraft.com/browse/FAL-3621)
